search/Property.cc

```diff
@@ -1163,10 +1180,18 @@ getProperty(Clock *clk,
     return PropertyValue(clk->period(), sta->units()->timeUnit());
   else if (stringEqual(property, "sources"))
```

Comment:
source_pins

Reply:
I didn't modify this property name in the PR, would you like me to change `sources` to `source_pins` in the next revision?

Reply:
Sorry, I thought you added it. Just leave it alone for compatibility (ie, put it back).

Reply:
Sure, will revert it